sql/moz-fx-data-shared-prod/mozilla_org/desktop_conversion_events/metadata.yaml

```diff
@@ -0,0 +1,17 @@
+friendly_name: Desktop Conversion Events
```

Comment:
I'd recommend copying the minimum set of metadata here, which for the matter at hand is just `workgroup_access`. In particular I don't think you should copy `references` as that's typically auto-generated metadata.

Reply:
got it, will fix that, thanks!